content/intro-to-storybook/vue/en/composite-component.md

```diff
@@ -248,21 +210,20 @@ However, sometimes the devil is in the details. A test framework that is explici
 
 In our case, we want our `TaskList` to render any pinned tasks **before** unpinned tasks that it is passed in the `tasks` prop. Although we have a story (`WithPinnedTasks`) to test this exact scenario; it can be ambiguous to a human reviewer that if the component **stops** ordering the tasks like this, it is a bug. It certainly won’t scream **“Wrong!”** to the casual eye.
 
-So, to avoid this problem, we can use Jest to render the story to the DOM and run some DOM querying code to verify salient features of the output.
+So, to avoid this problem, we can use Jest to render the story to the DOM and run some DOM querying code to verify salient features of the output. The nice thing about the story format is that we can simply import the story in our tests, and render it there!
 
-Create a test file called `tests/unit/TaskList.spec.js`. Here we’ll build out our tests that make assertions about the output.
+Create a test file called `src/components/TaskList.spec.js`. Here we’ll build out our tests that make assertions about the output.
 
 ```javascript
-
-//tests/unit/TaskList.spec.js
+// src/components/TaskList.spec.js
 import Vue from 'vue';
-import TaskList from '../../src/components/TaskList.vue';
-import { withPinnedTasksData } from '../../src/components/TaskList.stories';
+import { WithPinnedTasks } from './TaskList.stories';
 
 it('renders pinned tasks at the start of the list', () => {
-  const Constructor = Vue.extend(TaskList);
+  // render WithPinnedTasks story
+  const Constructor = Vue.extend(WithPinnedTasks);
   const vm = new Constructor({
-    propsData: { tasks: withPinnedTasksData },
+    propsData: WithPinnedTasks.args,
   }).$mount();
   const firstTaskPinned = vm.$el.querySelector('.list-item:nth-child(1).TASK_PINNED');
 
```

Comment:
@andre-brdoch when you're able update the wording to :
"Create a test file called `tests/unit/TaskList.spec.js`. Here we’ll build out our tests that make assertions about the output".

And the snippet should be:
```js
// tests/unit/TaskList.spec.js

import Vue from "vue";

import TaskList from "../../src/components/TaskList.vue";

import { WithPinnedTasks } from "../../src/components/TaskList.stories";

it("renders pinned tasks at the start of the list", () => {
  const Constructor = Vue.extend(TaskList); 
  const vm = new Constructor({
    propsData: WithPinnedTasks.args,
  }).$mount();
  const firstTaskPinned = vm.$el.querySelector(
    ".list-item:nth-child(1).TASK_PINNED"
  );
  // We expect the pinned task to be rendered first, not at the end
  expect(firstTaskPinned).not.toBe(null);
});

```

In React it's good to go and we could import directly the story and the associated template and the test would run fine. With Vue it's a bit different. We'll need to make a small compromise and inject the component directly. See the [issue](https://github.com/andre-brdoch/learnstorybook-v6-vue/issues/3) i raised in your repo for more context.